app/pages/BeelieversAuction/MintInfo.tsx

```diff
@@ -117,7 +117,7 @@ function MintAction({ isWinner, doRefund }: MintActionProps) {
 					onClick={handleRefund}
 				>
 					💰 Refund
-					<div className="text-small text-muted-foreground">
```

Comment:
why removing this class?

Reply:
then you should update it here as well, right?

Reply:
Moved it back as `text-muted-foreground ` since daisyUI not accepting other variables.